packages/teams-js/src/internal/dialogHelpers.ts

```diff
@@ -83,11 +89,19 @@ export function botUrlOpenHelper(
 }
 
 export function urlSubmitHelper(apiVersionTag: string, result?: string | object, appIds?: string | string[]): void {
-  ensureInitialized(runtime, FrameContexts.task);
+  // FrameContext content should not be here because dialog.submit can be called only from inside of a dialog (FrameContext task)
```

Comment:
Do we need to create a TODO to track?

Reply:
Also, do we know if Teams mobile has a ETA to fix it properly on their end?

Reply:
@jekloudaMSFT do you know about the ETA? I have created a work item here:
https://office.visualstudio.com/MetaOS/_workitems/edit/10134868

Reply:
Teams mobile already has a work item for this, I'll see if I can link ours to theirs